js/modal.js

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable id-length */
+/* eslint-disable no-magic-numbers */
+/* eslint-disable max-statements */
+/* eslint-disable func-style */
+/* eslint-disable max-len */
+/* eslint-disable sort-keys */
+
+const projectList = {
```

Comment:
Kindly fix this error:
- When I click on a link in your navbar menu, it's overlapping all the elements.
![Screen Shot 2021-08-04 at 16 32 48](https://user-images.githubusercontent.com/22258274/128257983-325ea7d0-b0d1-4e12-8e0d-e3c7bd3b87b0.png)